components/EmbedVideo.js

```diff
@@ -33,17 +50,17 @@ const VideoIframeContainer = ({
 }
 
 VideoIframeContainer.propTypes = {
-  youtubeVideo: PropTypes.string.isRequired,
+  videoUrl: PropTypes.string.isRequired,
   className: PropTypes.string.isRequired,
 }
 
-const YouTubeVideo = styled(VideoIframeContainer)`
+const EmbedVideo = styled(VideoIframeContainer)`
```

Comment:
Isn't it an `EmbeddedVideo` rather than `EmbedVideo`?

Reply:
Yes, I agree. I have pushed that name change now.